lib/decidim/vocdoni/admin_engine.rb

```diff
@@ -24,6 +25,7 @@ class AdminEngine < ::Rails::Engine
           resources :census, only: [:index, :create] do
             delete :destroy_all, on: :collection, as: :destroy_all
           end
+          resources :election_calendar, only: [:edit, :update]
```

Comment:
There's a weird thing with this URL that we should check out. I'm not sure how to actually solve it though. 

If you see how the URLs are now, we have: 

* /elections/new
* /elections/4/edit
* /elections/4/questions
* /elections/4/questions/new
* /elections/4/census
* /elections/4/election_calendar/4/edit
* /elections/4/publish_page

IMHO all make sense except the calendar page, as we have the ID two times (and as far as I see the second time is ignored): `/elections/4/election_calendar/4/edit`. Can we find a way of leaving out only `/elections/4/calendar`?